craft_parts/plugins/cmake_plugin.py

```diff
@@ -101,6 +99,7 @@ def get_build_commands(self) -> List[str]:
             f'"{self._part_info.part_src_subdir}"',
             "-G",
             f'"{options.cmake_generator}"',
+            "-DCMAKE_INSTALL_PREFIX=",
```

Comment:
please add a code comment on the effects or why this is here.

Reply:
Comment added for this variable and for `CMAKE_PREFIX_PATH` above.